.gitignore

```diff
@@ -28,3 +28,5 @@ cover/
 .cache/
 htmlcov/
 coverage.xml
+env/
+.vscode/
```

Comment:
Please remove the change to the .gitignore file since they are folder local do your computer. You can easily add it to your [global gitignore](https://davidwalsh.name/global-gitignore) instead.